aws-redshift-scheduledaction/aws-redshift-scheduledaction.json

```diff
@@ -56,18 +56,32 @@
         },
         "ScheduledActionType": {
             "type": "object",
-            "properties": {
-                "ResizeCluster": {
-                    "$ref": "#/definitions/ResizeClusterMessage"
+            "oneOf": [
```

Comment:
General question: can u elaborate more about why these schema changes are needed for the new test framework? Is it related to the json file format needed to run the V2 tests?

Reply:
This schema change due to a new testing case: `test_read_return_all_schema_properties_using_one_or_more_inputs`, which was defined as *"This asserts that one or more READ requests using one or more inputs files MUST return ALL schema returnable properties"*. ScheduledAction can only take one [*ScheduledActionType*](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-redshift/blob/8fe7877314239fca3fec52eda246215795d94471/aws-redshift-scheduledaction/aws-redshift-scheduledaction.json#L57-L69) at a time. Originally, we do not have this restriction in CFN Schema, so that we let the Front-End do the validation for us. However, due to the new case `test_read_return_all_schema_properties_using_one_or_more_inputs` I mentioned earlier, it required us to return *all* of them by default, which is impossible. Thus, I added a new `oneOf` restriction to make `test_read_return_all_schema_properties_using_one_or_more_inputs` pass.